dnf/package.py

```diff
@@ -38,6 +38,9 @@
 class Package(hawkey.Package):
     """ Represents a package. #:api """
 
+    DEBUGINFO_SUFFIX = "-debuginfo"  # :api
+    DEBUGSOURCE_SUFFIX = "-debugsource"  # :api
```

Comment:
@m-blaha asking for one more opinion, do you think these constants should be inside the `Package` class or outside, in the module? Seems a bit of a toss-up to me :slightly_smiling_face:

Reply:
I don't have strong opinion on this. Usually we define API constants on module level, but since these constants are tightly connected with Package class and not used out of this class, I tend to leave them in the Package namespace. Btw, do they need to by an API? In case they need to be an API, they should be documented in doc/api_package.rst

Reply:
I'm using them in a plugin, which means they need to be API, right?

Reply:
Yes, in that case they should be. If they are used strictly in Package context, I'm not against leaving them in Package namespace.

Reply:
I'll keep them inside the `Package` then, since they are used to find out if a package name ends with the given suffix.